src/main/java/nl/praegus/fitnesse/slim/fixtures/mockserver/MockServer.java

```diff
@@ -137,10 +138,11 @@ public void forwardRequestsOnToWithPath(String path, String target, String fwPat
             scheme = SocketAddress.Scheme.HTTPS;
         }
 
-        target = target.split(":")[1];
-        if (target.contains(":")) {
-            host = target.split(":")[0];
-            port = Integer.parseInt(target.split(":")[1]);
+        host = target.split("://")[1];
+
+        if (host.split(":").length == 2) {
+            port = Integer.parseInt(host.split(":")[1]);
+            host = host.split(":")[0];
         }
```

Comment:
115-119 en 141-145 zijn 2x hetzelfde. Zullen we die lostrekken in een methode die bijv een InetSocketAddress teruggeeft?

Reply:
Yep, daar ga ik nog even naar kijken